sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/Item.kt

```diff
@@ -0,0 +1,26 @@
+package co.nimblehq.sample.compose.ui.screens.home
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import co.nimblehq.sample.compose.model.UiModel
+import co.nimblehq.sample.compose.ui.theme.Dimension
+
+@Composable
+fun Item(
+    uiModel: UiModel,
+    onClick: (UiModel) -> Unit
+) {
+    Surface(
+        modifier = Modifier
+            .fillMaxWidth()
+            .clickable(onClick = { onClick(uiModel) })
+    ) {
+        Row(modifier = Modifier.padding(Dimension.SpacingNormal)) {
+            Text(text = uiModel.id.toString())
+        }
+    }
+}
```

Comment:
@chornerman I don't think we need a `Surface` here, Let's add the preview then we can see how was it look like. Here's some idea:
```suggestion
    Row(
        modifier = Modifier
            .fillMaxWidth()
            .padding(Dimension.SpacingNormal)
            .clickable(onClick = { onClick(uiModel) })
    ) {
        Text(text = uiModel.id.toString())
    }
}

@Preview(showBackground = true)
@Composable
fun ItemPreview() {
    val uiModel = UiModel(id = 1)
    Theme {
        Item(
            uiModel = uiModel,
            onClick = { uiModel.id }
        )
    }
}
```

Reply:
Oh, my bad 🙏 I got it now why you need to use Surface, We need to make the background
cover all item when clicked 🚀